presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -133,6 +133,9 @@ Accessors
 
     Returns the point value that is the mathematical centroid of a geometry.
 
+.. function:: ST_ConvexHull(Geometry) -> Geometry
+    Returns the convex hull of a geometry represents the minimum convex geometry that encloses all geometries within the set.
```

Comment:
This should have a newline before the description (see others for examples).

The sentence is not valid. I think you meant to say “a geometry that represents”

Reply:
For clarity, let's say "Returns the minimum convex geometry that encloses all input geometries."